koala/cogs/base/api.py

```diff
@@ -89,6 +106,113 @@ async def post_schedule_activity(self, activity_type, message, url, start_time,
 
         return build_response(CREATED, {'message': 'Activity scheduled'})
 
+    @parse_request
+    async def get_ping(self):
+        """
+        Get the latency of the bot
+        :return: The ping
+        """
+        return await core.ping(self._bot)
+
+    @parse_request()
+    async def get_support_link(self):
+        """
+        Get the support link of KoalaBot
+        :return: The support link
+        """
+        return core.support_link()
+
+    @parse_request
+    async def get_version(self):
+        """
+        Get the version of KoalaBot
+        :return: The version
+        """
+        return core.get_version()
+
+    @parse_request(raw_response=True)
```

Comment:
is there a reason why raw_response was needed? Could your instead return the dictionary {'message': 'Cog loaded'} which will the be parsed with the ok response?

Reply:
I'm not sure what you mean by parsed with the OK response 🤔 if it returns with OK status, doesn't that need a build_response?